text/0148-sampling-probability.md

```diff
@@ -0,0 +1,291 @@
+interest, after calculating a sample from a sample frame.
+
+- *Sample size*: the number of events with non-zero inclusion probability
+- *True population total*: the exact number of events in the frame
+- *Estimated population total*: the estimated number of events in the frame
+
+The sample size is known after it is calculated, but the size may or
+may not be known ahead of time, depending on the design.  The true
+population total cannot be inferred directly from the sample, but can
+(sometimes) be counted separately.  The estimated population total is
+the expected value of the true population total.
+
+### Adjusted sample count
+
+Following the model above, every event defines the notion of an
+_adjusted count_.
+
+- _Adjusted count_ is zero if the event was not selected for the sample
+- _Adjusted count_ is the reciprocal of its inclusion probability, otherwise.
+
+The adjusted count of an event represents the expected contribution to
+the estimated population total of a sample framer represented by the
+individual event.  As stated, the sample event's adjusted count is
+easily derived from the Horvitz-Thompson estimator of the population
+total, a general-purpose statistical estimator that applies to all
+_without replacement_ sampling designs.
+
+Assuming sample data is correctly computed, the consumer of sample
+data can treat every sample event as though an identical copy of
+itself has occurred _adjusted count_ times.  Every sample event is
+representative for adjusted count many copies of itself.
+
+There is one essential requirement for this to work.  The selection
+procedure must be _statistically unbiased_, a term meaning that the
+process is required to give equal consideration to all possible
+outcomes.
+
+### Encoding inclusion probability
+
+Some possibilities for encoding the inclusion probability, depending
+on the circumstances and the protocol, briefly discussed:
+
+1. Encode the adjusted count directly as a floating point or integer number in the range [0, +Inf).  This is a conceptually easy way to understand sampling because larger numbers mean greater representivity.
+2. Encode the inclusion probability directly as a floating point number in the range [0, 1).  This is typical of the Statsd format, where each line includes an optional probability.  In this context, the probability is commonly referred to as a "sampling rate".  In this case, smaller numbers mean greater representivity.
+3. Encode the negative of the base-2 logarithm of inclusion probability.  This restricts inclusion probabilities to powers of two and allows the use of small non-negative integers to encode power-of-two adjusted counts.
+4. Fold the adjusted count into the data.  This is appropriate when the data itself carries counts, such as for OTLP Metrics Sum and Histogram points encoded using delta aggregation temporality.  This may lead to rounding errors, when adjusted counts are not integer valued.
+
```

Comment:
Maybe mention that the 4th possibility is different from the first 3 as you loose information about the variance. Without additional information, you no longer know if the count is exact or represents an estimate, and if the sampling rate was the same for all increments or not.

Reply:
Done.